Update the Get Started tutorial for Go connector (#4193)
tarantool/doc: doc/how-to/getting_started_net_box.rst

Sample database configuration
-----------------------------

.. connectors_sample_db_config_start

This section describes the :ref:`configuration <configuration_file>` of a sample database that allows remote connections:

..  literalinclude:: /code_snippets/snippets/connectors/instances.enabled/sample_db/config.yaml
    :language: yaml
    :dedent:

-   The configuration contains one instance that listens for incoming requests on the ``127.0.0.1:3301`` address.
-   ``sampleuser`` has :ref:`privileges <authentication-owners_privileges>` to select and modify data in the ``bands`` space and execute the ``get_bands_older_than`` stored function. This user can be used to connect to the instance remotely.

> -   ``myapp.lua`` defines how data is stored in a database and includes a stored function.

Reviewer: ```suggestion
-   ``myapp.lua`` defines the data model and a stored function.
```
